packer/jenkins.json

```diff
@@ -4,6 +4,10 @@
     "region": "{{env `AWS_DEFAULT_REGION`}}"
   },
   "provisioners": [
+    {
+      "type": "shell",
+      "inline": ["sudo yum install -y perl-Digest-SHA.x86_64"]
```

Comment:
This is required by the CloudWatch role, yeah? Want to send a pull request there, or should I? Also, can we move this into the playbook so we can add a comment about why it’s present?

Reply:
It's required for the perl script to function on RHEL. Ubuntu appears to include the necessary perl packages by default. In my mind, this should be part of the packer build rather than in the ansible playbook since it would be a standard part of the image, but I can also see where it would be considered more on the cloudwatch role only. I guess to me it was part of the base image since it's specific to RHEL.